code/modules/lighting/lighting_atom.dm

```diff
@@ -46,21 +46,13 @@
 		T.reconsider_lights()
 	return ..()
 
-/atom/movable/Move()
-	var/turf/old_loc = loc
+/atom/Entered(atom/movable/obj, atom/prev_loc)
```

Comment:
Hm. I'm not sure this is ideal, because sending something to nullspace would mean atom/Entered() is not called.
Then again, Move() might not be called either.